crates/primitives/starknet/src/tests/block.rs

```diff
@@ -89,7 +89,7 @@ fn test_transactions_hashes_from_hashes() {
     let vec_hashes: Vec<H256> = transactions.iter().map(|tx| tx.hash).collect();
     let hashes = BoundedVec::<H256, MaxTransactions>::try_from(vec_hashes).unwrap();
 
-    let block = Block::new(header, BlockTransactions::Hashes(hashes.clone()));
+    let block = Block::new(header, hashes.clone());
```

Comment:
How does this compiles?

Reply:
Oh it doesn't. This file is not part of the project.
Add `pub mod block;` in `crates/primitives/starknet/src/tests/mod.rs` and warning will be raised